helm/oauth2-proxy/README.md

```diff
@@ -1,6 +1,6 @@
 # oauth2-proxy
 
-![Version: 2.14.2](https://img.shields.io/badge/Version-2.14.2-informational?style=flat-square) ![AppVersion: 7.7.0](https://img.shields.io/badge/AppVersion-7.7.0-informational?style=flat-square)
+![Version: 3.1.0](https://img.shields.io/badge/Version-3.1.0-informational?style=flat-square) ![AppVersion: 7.7.1](https://img.shields.io/badge/AppVersion-7.7.1-informational?style=flat-square)
```

Comment:
I assume that was forgotten in a previous PR, as I see no changes related to version here.